templates/garden/nightly/bert-mnli.libsonnet

```diff
@@ -59,11 +59,29 @@ local tpus = import "../../tpus.libsonnet";
       '--eval_batch_size=32',
     ],
   },
+  local v2_32 = {
+    accelerator: tpus.v2_32,
+    command+: [
+      '--train_batch_size=128',
+      '--eval_batch_size=128',
```

Comment:
BERT should be able to run with a batch size of 512

Reply:
Fixed these batch sizes.